### PR TITLE
Add MaxLimit

### PR DESCRIPTION
**Pull request by gillesfabio**

### CODE REVIEW

**Comment by thoas** [comment]
we shouldn't put a ``DefaultMaxLimit`` I think

File: utils.go [hunk 1]
```diff
@@ -32,6 +32,10 @@ func GetLimitFromRequest(request *http.Request, options *Options) int64 {
 		limit, err = strconv.ParseInt(requestLimit, 10, 64)
 		if err != nil {
 			limit = options.DefaultLimit
+
+		}
+		if limit > options.MaxLimit {
```
Comment (thoas) [inline utils.go]: ``if options.MaxLimit > 0 && limit > options.MaxLimit``

**Comment by gillesfabio** [comment]
Yes, only optional could be better.

**Comment by thoas** [comment]
yep :)

**Comment by gillesfabio** [comment]
Done.

**Pull request merged**